charger/easee.go

```diff
@@ -685,12 +746,36 @@ func (c *Easee) CurrentPower() (float64, error) {
 
 var _ api.ChargeRater = (*Easee)(nil)
 
+func (c *Easee) RequestLifetimeEnergyUpdate() (err error) {
```

Comment:
As all callers are ignoring the possible error, just remove it from the signature.